src/jit/codegencommon.cpp

```diff
@@ -8935,6 +8935,12 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
     // This is the end of the OS-reported prolog for purposes of unwinding
     compiler->unwindEndProlog();
 
+    // If there is no PSPSym (CoreRT ABI), we are done.
+    if (compiler->IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
It may be better to check `if (compiler->lvaPSPSym == BAD_VAR_NUM)` so that it looks exactly same as the x64 path.

Reply:
Fixed